From c693bab827ea14b51e2cc15942d9067c3d2dacbd Mon Sep 17 00:00:00 2001
Message-Id: <c693bab827ea14b51e2cc15942d9067c3d2dacbd.1659372189.git.thomas@gllm.fr>
In-Reply-To: <5b01e96eabd14521cfeb33b5ac11495b77f894cc.1659372189.git.thomas@gllm.fr>
References: <5b01e96eabd14521cfeb33b5ac11495b77f894cc.1659372189.git.thomas@gllm.fr>
From: Thomas Guillem <thomas@gllm.fr>
Date: Mon, 25 Jul 2022 14:05:27 +0200
Subject: [PATCH 12/19] contrib: rav1e: fix build on Android NDK25

I have absolutely no idea where "lgcc" is added between rav1e/cargo/cargo-c.
---
 contrib/src/rav1e/rules.mak | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/contrib/src/rav1e/rules.mak b/contrib/src/rav1e/rules.mak
index 51735454de..885a8464ea 100644
--- a/contrib/src/rav1e/rules.mak
+++ b/contrib/src/rav1e/rules.mak
@@ -33,4 +33,8 @@ endif
 
 .rav1e: rav1e .cargo
 	+cd $< && $(CARGOC_INSTALL) --no-default-features $(RAV1E_FEATURES)
+# No gcc in Android NDK25
+ifdef HAVE_ANDROID
+	sed -i -e 's/ -lgcc//g' $(PREFIX)/lib/pkgconfig/rav1e.pc
+endif
 	touch $@
-- 
2.35.1

